esp-box/esp-box.c

```diff
@@ -370,3 +371,22 @@ bool bsp_button_get(const bsp_button_t btn)
 {
     return !(bool)gpio_get_level(btn);
 }
+
+bool bsp_touch_button_get(void)
+{
+    uint8_t home_btn_val = 0x00;
+
+    assert(indev);
+    lv_indev_drv_t *indev_drv = indev->driver;
+    assert(indev_drv);
+    lvgl_port_touch_ctx_t *touch_ctx = (lvgl_port_touch_ctx_t *)indev_drv->user_data;
+    assert(touch_ctx);
```

Comment:
This is not working! Please put `static esp_lcd_touch_handle_t tp;` out to top of file and then use it like this:
```
    uint8_t home_btn_val = 0x00;
    assert(tp);

    esp_lcd_touch_get_button_state(tp, 0, &home_btn_val);
```